crates/hotshot/src/traits/networking/web_server_network.rs

```diff
@@ -927,6 +1058,44 @@ impl<
                 }
             }
 
+            ConsensusIntentEvent::CancelPollForVIDVotes(view_number) => {
+                let mut task_map = self.inner.vid_vote_task_map.write().await;
+
+                if let Some((_, sender)) = task_map.remove_entry(&(view_number)) {
+                    // Send task cancel message to old task
+
+                    // If task already exited we expect an error
+                    let _res = sender
+                        .send(ConsensusIntentEvent::CancelPollForVIDVotes(view_number))
+                        .await;
+                }
+            }
+
+            // ConsensusIntentEvent::CancelPollForVIDDisperse(view_number) => {
+            //     let mut task_map = self.inner.vid_disperse_task_map.write().await;
+
+            //     if let Some((_, sender)) = task_map.remove_entry(&(view_number)) {
+            //         // Send task cancel message to old task
+
+            //         // If task already exited we expect an error
+            //         let _res = sender
+            //             .send(ConsensusIntentEvent::CancelPollForVIDDisperse(view_number))
+            //             .await;
+            //     }
+            // }
+
+            // ConsensusIntentEvent::CancelPollForVIDCertificate(view_number) => {
+            //     let mut task_map = self.inner.vid_cert_task_map.write().await;
+
+            //     if let Some((_, sender)) = task_map.remove_entry(&(view_number)) {
+            //         // Send task cancel message to old task
+
+            //         // If task already exited we expect an error
+            //         let _res = sender
+            //             .send(ConsensusIntentEvent::CancelPollForVIDCertificate(view_number))
+            //             .await;
+            //     }
+            // }
```

Comment:
Nit: Can we delete this code if we aren't going to use it?

Reply:
Sure can ! I forgot to remove this